modules/virt-cnv-virt-alerts.adoc

```diff
@@ -0,0 +1,487 @@
+[source,terminal]
+----
+$ oc -n $NAMESPACE logs <pod-name> |grep lead
+{"component":"virt-operator","level":"info","msg":"Attempting to acquire leader status","pos":"application.go:400","timestamp":"2021-11-30T12:15:18.635387Z"}
+I1130 12:15:18.635452       1 leaderelection.go:243] attempting to acquire leader lease <namespace>/virt-operator...
+I1130 12:15:19.216582       1 leaderelection.go:253] successfully acquired lease <namespace>/virt-operator
+{"component":"virt-operator","level":"info","msg":"Started leading","pos":"application.go:385","timestamp":"2021-11-30T12:15:19.216836Z"}
+----
+
+*Non-leader pod example:*
+[source,terminal]
+----
+$ oc -n $NAMESPACE logs <pod-name> |grep lead
+{"component":"virt-operator","level":"info","msg":"Attempting to acquire leader status","pos":"application.go:400","timestamp":"2021-11-30T12:15:20.533696Z"}
+I1130 12:15:20.533792       1 leaderelection.go:243] attempting to acquire leader lease <namespace>/virt-operator...
+----
+
+.Resolution
+
+There are several reasons for no virt-operator pod holding the leader lease, despite one or more virt-operator pods being in `Ready` state. Identify the root cause and take appropriate action.
+
+Otherwise, open a support issue and provide the information gathered in the troubleshooting process.
+
+//NoReadyVirtController Alert
+[id="NoReadyVirtController_{context}"]
+== NoReadyVirtController alert
+
+.Description
+
+The virt-controller monitors virtual machine instances (VMIs). The virt-controller also manages the associated pods by creating and managing the lifecycle of the pods associated with the VMI objects.
+
+A VMI object always associates with a pod during its lifetime. However, the pod instance can change over time because of VMI migration.
+
+This alert occurs when detection of no ready virt-controllers occurs for five minutes.
+
+.Reason
+
+If the virt-controller fails, then VM lifecycle management completely fails. Lifecycle management tasks include launching a new VM instance or shutting down an existing VM instance.
+
+.Troubleshoot
+
+. Modify the environment variable `NAMESPACE`.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+----
+
+. Run this command:
++
+[source,terminal]
+----
+oc get deployment -n $NAMESPACE virt-controller -o jsonpath='{.status.readyReplicas}'
+----
+
+.Resolution
+
+. Check the virt-controller deployment's status to find out more information.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get deploy virt-controller -o yaml
+
+oc -n $NAMESPACE describe deploy virt-controller
+----
+
+. Get the virt-controller pods.
++
+[source,terminal]
+----
+get pods -n $NAMESPACE |grep virt-controller
+----
+
+. Inspect the logs for each virt-controller.
++
+[source,terminal]
+----
+oc logs -n $NAMESPACE <virt-controller-XYZ>
+----
+
+. Check if there are issues with the nodes, such as if the nodes are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
```

Comment:
Like the `Resolution` sections for other alerts above, it would be good to clarify here what all information should be added to support case, in case it comes to that.

Reply:
@yuhaohaoyu Can you please help here?

Reply:
@yuhaohaoyu What information would be useful for a user to send in a support ticket?

In other places in these alerts, we've said "Open a support issue and provide the information gathered in the troubleshooting process."

Since this question/comment has come up several times, I'm hoping for some clarity on what is important to tell a user.

Reply:
Or @kbidarkar would you have any insight on what information would be useful for a user to send in a support ticket? and if saying "Open a support issue and provide the information gathered in the troubleshooting process" is sufficient?

Reply:
"Open a support issue and provide the information gathered in the troubleshooting process" - this sounds good for me. About troubleshooting - see my previous comment